go.mod

```diff
@@ -591,6 +581,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require github.com/smartystreets/goconvey v1.7.2 // indirect
```

Comment:
What did we add? Can we roll up into the above required indirect block if this is needed?

Reply:
let me check :eye: